package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "videojs-generator-verify",
+  "version": "0.0.0",
+  "description": "A tool to verify that a videojs-generate-plugin project is ready for publish.",
+  "main": "src/index.js",
+  "bin": {
+    "vjsverify": "src/cli.js",
+    "videojs-verify": "src/cli.js",
+    "videojs-generator-verify": "src/cli.js"
+  },
+  "keywords": [],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/videojs/videojs-generator-verify"
+  },
+  "author": "Brandon Casey <brandonocasey@gmail.com>",
```

Comment:
Should this be "Brightcove, Inc."? I'm not sure it matters since this is under the Video.js org, though.

Reply:
🤷‍♂️